components/src/dynamo/profiler/utils/dgd_generation.py

```
@@ -22,438 +22,268 @@
-def _generate_mocker_config_with_planner(
-    args,
-    cm_mount_path: str,
-    config_map_obj: Optional[dict],
-    planner_dict: dict,
-) -> list[dict] | dict:
-    """Generate mocker DGD config with planner for testing purposes.
+def _build_planner_config(
+    dgdr,
+    best_prefill_mapping,
+    best_decode_mapping,
+    output_dir: str | None,
```

Ruff (0.15.1)

[warning] Unused function argument: `output_dir`

(ARG001)